astro/src/components/download/DownloadWidget.astro

```diff
@@ -0,0 +1,624 @@
+      </p>
+      <p class="mb-4">
+        <strong>Step { step++ }:</strong> Download the Docker files
+      </p>
+      <ul class="flex mb-4">
+        <li class="group active"><a data-tab-button href="#docker-compose-download-nix" class="block border-b-2 border-transparent font-bold px-7 py-3 text-xs text-slate-400 group-[.active]:border-indigo-400 group-[.active]:text-white">Linux/Unix/macOS</a></li>
+        <li class="group"><a data-tab-button href="#docker-compose-download-windows" class="block border-b-2 border-transparent flex-grow px-7 py-3 text-xs text-slate-400 group-[.active]:border-indigo-400 group-[.active]:text-white">Windows</a></li>
+      </ul>
+      <div data-tab-content id="docker-compose-download-nix">
+        <Code>
+          curl -fsSL https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/docker-compose.yml > docker-compose.yml && \
+          curl -fsSL https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/.env > .env
+        </Code>
+      </div>
+      <div data-tab-content id="docker-compose-download-windows" class="hidden">
+        <Code>
+          Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/docker-compose.yml -OutFile docker-compose.yml
+          Invoke-WebRequest -UseBasicParsing https://raw.githubusercontent.com/FusionAuth/fusionauth-containers/main/docker/fusionauth/.env -OutFile .env
+        </Code>
+      </div>
+      { kickstartEnabled &&
+          <p class="mb-2 mt-12">
+            <strong>Step { step++ }:</strong> Download the Kickstart files needed for this project
+          </p>
+          <p class="mb-4">
+            <em class="text-xs">NOTE: The Docker Compose configuration loads these Kickstart files automatically.</em>
+          </p>
+          <Kickstart env={false} includeNix={true} includeWindows={true} name={kickstartName} section="docker-compose-kickstart"/>
+      }
+      <p class="mb-6 mt-12">
+        <strong>Step { step++ }:</strong> Start the Docker Compose containers
+      </p>
+      <Code>
+        docker-compose up
+      </Code>
+      <p class="mb-6">
+        Then open <a href="http://localhost:9011" class={anchorClass}>http://localhost:9011</a> in your browser.
+      </p>
+    </div>
+    <div data-tab-content id="docker-vanilla" class="hidden m-8" data-reset={step = 1}>
+      <h2 class="mb-4 text-2xl">Docker Vanilla</h2>
+      <p class="mb-6">
+        Use the following Docker commands to start just FusionAuth without a database. You'll need to install and configure a database manually. Feel free to open an issue in <a href="https://github.com/FusionAuth/fusionauth-issues" class={anchorClass}>GitHub</a> if you find any issues.
```

Comment:
What's the context where you'd want that? Do you want to warn folks that they should set the environment vars beforehand (or be prepared to go through maintenance mode)?

Reply:
This is parity with the current brochure site download widget.

I definitely can see instances where people want to run the Docker container by itself and this seems to cover that you'll need a database. And even with the Docker Compose, you still need to go through maintenance mode, so really it's just about the same level of effort if you already have a database running on the hose machine.